libp2p/7-properties.md

```diff
@@ -190,3 +190,19 @@ For now, we use [protobuf](https://github.com/google/protobuf) for all protocol
 ### 7.5.6 Secure Communications
 
 The wire protocol is -- of course -- wrapped with encryption. We use cyphersuites similar to TLS. This is explained further in the [network spec](./#encryption).
+
+### 7.5.7 Protocol Multicodecs
+
+Here, we present a table with the multicodecs defined for each IPFS protocol that has a wire componenent. This list may change overtime and currently exists as a guide for implementation
+
+| protocol | multicodec |
+| :------: | :--------: |
+| secio    | /ipfs/secio/1.0.0 |
```

Comment:
Yeap, I beleive that is correct. from this doc: https://gist.github.com/whyrusleeping/c1b53200dc2072564478f1b9e3671371#gistcomment-1743999